src/neo/Wallets/NEP6/NEP6Account.cs

```diff
@@ -83,5 +83,30 @@ public bool VerifyPassword(string password)
                 return false;
             }
         }
+
+        public bool ChangePassword(string password_old, string password_new)
+        {
+            if (WatchOnly) return true;
+            if (nep2key == null)
+            {
+                if (key == null)
+                {
+                    return true;
+                }
+            }
+            else
+            {
+                if (!VerifyPassword(password_old))
+                {
+                    return false;
+                }
+                else if (key == null)
+                {
+                    key = wallet.DecryptKey(nep2key);
+                }
```

Comment:
You don't need to decrypt the key twice. `VerifyPassword()` is the first time, and `DecryptKey()` is the second time.

Reply:
> You don't need to decrypt the key twice. `VerifyPassword()` is the first time, and `DecryptKey()` is the second time.

Thx! Have modified correspondingly:)